controllers/clusterenvironment_controller.go

```diff
@@ -400,12 +402,59 @@ func (r *ClusterEnvironmentReconciler) updateClusterEnvironmentStatus(ctx contex
 	ce.Status.State = cs.State
 	meta.SetStatusCondition(&ce.Status.Conditions, cs.Condition())
 }
+func (r *ClusterEnvironmentReconciler) getRelatedClusterEnvironments(ctx context.Context, req ctrl.Request, envname string) ([]primazaiov1alpha1.ClusterEnvironment, error) {
+	cee := primazaiov1alpha1.ClusterEnvironmentList{}
+
+	if err := r.List(ctx, &cee, &client.ListOptions{Namespace: req.Namespace}); err != nil {
+		return nil, err
+	}
+
+	ff := r.filterClusterEnvironments(envname, cee.Items)
+	return ff, nil
+}
```

Comment:
A line to be left after each function. L414 and L404

Reply:
okay